src/OpenLoco/src/Vehicles/VehicleBody.cpp

```diff
@@ -100,13 +100,13 @@ namespace OpenLoco::Vehicles
         }
 
         const auto* vehicleObject = getObject();
-        int32_t var_05 = vehicleObject->carComponents[bodyIndex].var_05;
-        if (var_05 == 0)
+        int32_t emitterHorizontalPos = vehicleObject->carComponents[bodyIndex].emitterHorizontalPos;
+        if (emitterHorizontalPos == 0)
         {
             return;
         }
 
-        var_05 -= 0x80;
+        emitterHorizontalPos -= 0x80;
```

Comment:
convert uint8_t to int8_t? For a future PR.

Reply:
you can't do that due to a bunch of odd steps that were taken with this variable where changing its type would change the calc.

Reply:
If all the users of emitterHorizontalPos are implemented, I think it could be changed to an int8_t on loading the object and skip all the offsets during simulation.